GarbageCollector section is not clear

mburke5678 opened the pull request:
https://bugzilla.redhat.com/show_bug.cgi?id=1704598

sunilcio commented:
LGTM

sunilcio commented:
@mburke5678 I just checked your question in bz. Garbage collection for containers is enabled by default. For image garbage collection, user need to manually enable it. So we need to modify the statement to reflect that in image garbage collection.

kalexand-rh commented on admin_guide/garbage_collection.adoc:
```diff
@@ -15,9 +15,9 @@ toc::[]
 The {product-title} node performs two types of garbage collection:
 
 * xref:container-garbage-collection[Container garbage collection] Removes
```
I'd add a : after the xref.

mburke5678 commented:
@kalexand-rh Thanks for the tips. I moved the notes and they do look better. 

@sunilcio I removed the references to the default values for the image garbage collection parameters based on Kathryn's comment above. It seems non-intuitive to say a parameter is not default but has default values. (It does seems that the parameters do have defaults.). Let me know if you think I should put them back.

mburke5678 commented:
/cherrypick enterprise-3.11

sunilcio commented:
I agree, looks better now. LGTM
